download-build/action.yml

```
@@ -61,10 +64,16 @@ runs:
         JFROG_DL_BUILD: ${{ github.event.repository.name }}/${{ inputs.build-number }}
         JFROG_DL_EXCLUSIONS: ${{ inputs.exclusions }}
         JFROG_DL_REMOTE_REPO: ${{ inputs.remote-repo }}
-      run: jfrog rt download
-        --fail-no-op ${{ steps.define-download-options.outputs.jfrog_dl_options }}
-        --build "${JFROG_DL_BUILD}"
-        "${JFROG_DL_REMOTE_REPO}"
+        groupIdPath: ${{ inputs.filter }}
```

Is it a groupIdPath or a filter? I would stay consistent with the naming, for instance filter.
```suggestion
        filter: ${{ inputs.filter }}
```